CLI/test/ressources/CopyBrokenFormat/output_expected/error.txt

```diff
@@ -1,8 +1,9 @@
 ReturnCode=ParsingDiagnostics_
-6 errors in "input\COMPCOPY.pgm":
+7 errors in "input\COMPCOPY.pgm":
 Line 10[8,22] <27, Error, Syntax> - Error in copy 'YBRKCPY' at line 1 : Syntax error : Line exceed 80 chars
 Line 10[8,22] <27, Error, Syntax> - Error in copy 'YBRKCPY' at line 2 : Syntax error : Line exceed 80 chars
 Line 10[8,22] <27, Error, Syntax> - Error in copy 'YBRKCPY' at line 3 : Syntax error : Literal is not correctly delimited.
 Line 10[8,22] <27, Error, Syntax> - Error in copy 'YBRKCPY' at line 5 : Syntax error : Line exceed 80 chars
-Line 10[8,22] <27, Error, Syntax> - Error in copy 'YBRKCPY' at line 3 : Syntax error : Invalid VALUE clause
 Line 10[8,22] <27, Error, Syntax> - Error in copy 'YBRKCPY' at line 4 : Syntax error : extraneous input '00900005' expecting {separator, statement starting keyword, keyword}
+Line 10[8,22] <32, Error, Syntax> - Error in copy 'YBRKCPY' at line 3 : A Cobol statement cannot be across 2 sources files (eg. Main program and a COPY)
```

Comment:
This is because we now have a `MissingToken` among the consumed tokens of the CE. I've updated the code of `CalculateIsAcrossSourceFile()` to handle this.